pkg/event/component_test.go

```diff
@@ -267,6 +267,26 @@ func TestEventToMapLogonError(t *testing.T) {
 
 }
 
+func TestEventToMapLogonErrorLocked(t *testing.T) {
+	var etype int8 = 1
+
+	var event *fb.Event
+	{
+		var builder = flatbuffers.NewBuilder(0)
+		var error = builder.CreateString("user_temporarily_disabled")
+		fb.EventStart(builder)
+		fb.EventAddType(builder, etype)
+		fb.EventAddError(builder, error)
+		var eventOffset = fb.EventEnd(builder)
+		builder.Finish(eventOffset)
+		event = fb.GetRootAsEvent(builder.FinishedBytes(), 0)
+	}
+
+	var m = eventToMap(event)
+	assert.Equal(t, "TEMPORARILY_LOCKED", m[database.CtEventType])
```

Comment:
Please add a test with a different error and one with a missing error (to ensure that f["error"] is not failing)

Reply:
Good suggestion; 
The missing error would correspond to TestEventToMapLogonError
I will add the one with diff error

Reply:
I ran the tests to check coverage: I think the different error is already check